draft-ietf-tls-ctls.md

```diff
@@ -45,12 +45,12 @@ informative:
 
 --- abstract
 
-This document specifies a "compact" version of TLS and DTLS. It is logically
+This document specifies a "compact" version of TLS 1.3 and DTLS 1.3. It is logically
```

Comment:
I know the title says "1.3", but the text actually specifies an (ostensibly) version-independent framework.  So while "cTLS 1.3" is the only version that's defined today, I'm not sure whether this change is strictly correct.

Reply:
OK, and this document doesn't specify a compact version of TLS 1.4 ... at least not explicitly.  I'm fine with this change.